packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -273,15 +273,7 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
       !isOpen && isAnimating && isOnRightSide && AnimationClassNames.slideRightOut40,
       focusTrapZoneClassName
     ],
-    commands: [
-      classNames.commands,
-      {
-        marginTop: 18
```

Comment:
I think the marginTop is still needed. That shouldn't be an issue, right @phkuo?

Reply:
perhaps same with marginLeft as well

Reply:
If it's on a flex parent and not a flex child, then yes, margins are fine.

Reply:
Cool that should fix the screener regression